charts/opentelemetry-collector/templates/_helpers.tpl

```diff
@@ -110,3 +114,47 @@ Return if ingress is stable.
 {{- define "ingress.isStable" -}}
   {{- eq (include "ingress.apiVersion" .) "networking.k8s.io/v1" -}}
 {{- end -}}
+
+
+{{- define "opentelemetry-collector.daemonset-podAnnotations" -}}
+{{- if .Values.podAnnotations }}
+{{- toYaml .Values.podAnnotations }}
+{{- else if .Values.agentCollector.podAnnotations }}
+{{- toYaml .Values.agentCollector.podAnnotations }}
+{{- end }}
+{{- end }}
+
+{{- define "opentelemetry-collector.deployment-podAnnotations" -}}
+{{- if .Values.podAnnotations }}
+{{- toYaml .Values.podAnnotations }}
+{{- else if .Values.standaloneCollector.podAnnotations }}
+{{- toYaml .Values.standaloneCollector.podAnnotations }}
+{{- end }}
+{{- end }}
+
+{{- define "opentelemetry-collector.daemonset-podLabels" -}}
+{{- if .Values.podLabels }}
+{{- toYaml .Values.podLabels }}
+{{- else if .Values.agentCollector.podLabels }}
+{{- toYaml .Values.agentCollector.podLabels }}
+{{- end }}
+{{- end }}
+
+{{- define "opentelemetry-collector.deployment-podLabels" -}}
+{{- if .Values.podLabels }}
+{{- toYaml .Values.podLabels }}
+{{- else if .Values.standaloneCollector.podLabels }}
+{{- toYaml .Values.standaloneCollector.podLabels }}
+{{- end }}
+{{- end }}
+
+{{- define "opentelemetry-collector.annotations" -}}
+{{- if or .Values.annotations .Values.standaloneCollector.annotations}}
+annotations:
+{{- if .Values.annotations }}
+  {{ toYaml .Values.annotations }}
```

Comment:
Shouldn't it be?
```suggestion
{{- toYaml .Values.annotations | nindent 2 }}
```

Otherwise I think only first line will be aligned correctly

Reply:
I think it was working bc I didn't have the leading `-` that would remove the whitespace, but explicitly using `nindent` is better.